pwa/src/templates/wooItemDetailTemplate/WOOItemDetailTemplate.tsx

```diff
@@ -100,6 +101,55 @@ export const WOOItemDetailTemplate: React.FC<WOOItemDetailTemplateProps> = ({ wo
     }
   };
 
+  const getName = (name: string) => {
+    
+    const formattedName = name.replace(/_/g, ' ')
+    console.log(formattedName);
+    
+    switch (_.upperFirst(formattedName)) {
+      case "Bevindingen":
+        return t("Findings");
+      case "Conclusies":
+        return t("Conclusions");
+      case "Functiebenaming":
+        return t("Job title");
+      case "Gedraging":
+        return t("Behavior");
+      case "Onderdeel taak":
+        return t("Part of task");
+      case "Oordeel":
+        return t("Judgement");
+      case "Opdrachtgever":
+        return t("Client");
+      case "Organisatieonderdeel":
+        return t("Organizational unit");
+      default:
+        return t(_.upperFirst(formattedName));
+    }
+  };
+
+  function isDate(str: string) {
+    var regex = /^(\d{4})-(\d{2})-(\d{2})(?:T.*)?$/;
+    var match = str.match(regex);
+    if (!match) {
+      return false;
+    }
+    var year = parseInt(match[1], 10);
+    var month = parseInt(match[2], 10);
+    var day = parseInt(match[3], 10);
+    if (month < 1 || month > 12) {
+      return false;
+    }
+    var daysInMonth = [31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31];
+    if ((year % 4 === 0 && year % 100 !== 0) || (year % 400 === 0)) {
+      daysInMonth[1] = 29;
+    }
+    if (day < 1 || day > daysInMonth[month - 1]) {
+      return false;
+    }
+    return true;
+  }

```

Comment:
This can become much more simpler by using default javascript date functionality

```suggestion
  function isDate(str: string) {
    const date = new Date(str);
    return !isNaN(date);
  }
```

This works because when `new Date()` receives something that is not a valid date, it will return a `Invalid Date` object which is a `NaN` value.
That value can be checked with the built in `isNan()` function, which returns a boolean.